# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1424534951 18000
#      Sat Feb 21 11:09:11 2015 -0500
# Node ID 343ca73b1507a18ab6f55d9b61ebf5afc9b2ecfe
# Parent  fd4a2df4694eb23a64ebbb97c6aa9c34956efe03
allow rules that make symbolic links to be silenced

* src/Makefile.am (octave-cli-$(version)$(EXEEXT),
octave-gui-$(version)$(EXEEXT)): Allow rules to be silenced.

diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -160,17 +160,17 @@ remove-version-links:
 	done
 
 .PHONY: make-version-links remove-version-links
 
 ## We need these file names in the build tree because the wrapper
 ## program (main.cc) will try to invoke the versioned binaries.
 
 octave-cli-$(version)$(EXEEXT): octave-cli$(EXEEXT)
-	rm -f $@
+	$(AM_V_GEN)rm -f $@ && \
 	$(LN_S) $< $@
 
 octave-gui-$(version)$(EXEEXT): octave-gui$(EXEEXT)
-	rm -f $@
+	$(AM_V_GEN)rm -f $@ && \
 	$(LN_S) $< $@
 
 CLEANFILES = \
   $(OCTAVE_VERSION_LINKS)
